app/models/user.rb

```diff
@@ -3,8 +3,6 @@
 class User < ApplicationRecord
   include PgSearch::Model
 
-  validates :gravity_user_id, presence: true, uniqueness: true
```

Comment:
@Gon4ar5 removing this check allows admins to manually create submissions without a user AND without a name/email/phone. That causes the `/submissions` and `/submission/[:id]` routes to fail with the following error:

```
NoMethodError: undefined method `user_detail' for #<Hyperclient::Resource:0x00007fc2055ed6d8>
  from hyperclient (1.0.1) lib/hyperclient/resource.rb:118:in `method_missing'
  from app/models/user.rb:40:in `user_detail'
  from app/models/user.rb:32:in `email'
  from app/views/admin/submissions/_submission.html.erb:22
  ...
  from app/views/admin/submissions/_submission.html.erb:3
 ...
  from app/views/admin/submissions/index.html.erb:91
```

We should patch this by reintroducing validation for `presence` if name/email/phone are not provided.

Reply:
@Gon4ar5 we removed the `uniqueness` check here, but we also need to remove that from the [database schema](https://github.com/artsy/convection/blob/ae17e282c887d75e4d86d5ab83e6d43a8e14b7df/db/schema.rb#L235-L237).